app/src/main/java/com/fs_sournary/weather/utils/BindingUtils.java

```diff
@@ -0,0 +1,30 @@
+package com.fs_sournary.weather.utils;
+
+import android.databinding.BindingAdapter;
+import android.databinding.ObservableBoolean;
+import android.support.v4.widget.SwipeRefreshLayout;
+
+/**
+ * Created by fs-sournary.
+ * Date on 11/17/17.
+ * Description:
+ */
+
+public class BindingUtils {
```

Comment:
WARNING: Utility classes should not have a public or default constructor.

Reply:
Em không khởi tạo constructor ở class này.